migrations/1616528363530-add_cashbox_batches.js

```diff
@@ -0,0 +1,26 @@
+var db = require('./db')
+
+exports.up = function (next) {
+  var sqls = [
+    `create table cashbox_batches (
+      id serial PRIMARY KEY,
```

Comment:
I don't think we're using serial IDs on any table, uuid.v4() is probably better here.